backend/config/routes.rb

```diff
@@ -8,6 +8,7 @@
 
     scope "/v1" do
       resources :books
+      resources :reading_lists
```

Comment:
I see only `create, update and destroy` methods in your controller. FYI that you can reduce the methods in your resources via

```
resources :reading_lists, only: %i[create update destroy]
```

This would reduce the noise when you do `rails routes` and ensure your API does not offer routes that are not serviced